async-query-core/src/test/java/org/opensearch/sql/spark/flint/operation/FlintIndexOpVacuumTest.java

```diff
@@ -66,6 +63,34 @@ public void setUp() {
             asyncQueryScheduler);
   }
 
+  // Helper method to create FlintIndexMetadata with latest ID
+  private static FlintIndexMetadata createFlintIndexMetadataWithLatestId() {
```

Comment:
Why did we extract?

Reply:
Not critical, but from my perspective, it is easier to read without extraction in this case, because the constant name provide the overview, and method name just repeat it. From reader's perspective, they'd need to jump to this method when they want to check what is actually done. (Constant definition is already working like an extracted method)